packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -380,6 +380,13 @@ const selectInputProps = computed(() => {
           <slot name="optionsEnd"></slot>
         </div>
       </div>
+      <div
+        v-if="props.listDescription"
+        :aria-description="props.listDescription"
```

Comment:
Shouldn't the aria-description rather be set on the list/flyout itself?

Reply:
@larsrickert I integrated the attribute in the headless composable.